sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -100,6 +100,7 @@ export class ProgramInfoRequestController {
     result.studentStudyStartDate = application.data.studystartDate;
     result.studentStudyEndDate = application.data.studyendDate;
     result.programYearId = application.programYear.id;
+    result.activeProgramYear = application.programYear.active;
```

Comment:
When I read activeProgramYear it gave me a quick impression of a year instead of flag. Can we rename the variable to something like  isProgramYearActive or programYearActive ?